src/utils/urlPaths.ts

```diff
@@ -27,15 +27,15 @@ export enum UrlPathTemplate {
   Logout = '/logout',
   MyPage = '/my-page',
   MyPageMessages = '/my-page/messages',
-  MyPageMyFieldAndBackground = '/my-page/my-profile/background',
+  MyPageMyFieldAndBackground = '/my-page/profile/background',
   MyPageMyMessages = '/my-page/messages/my-messages',
   MyPageMyMessagesRegistration = '/my-page/messages/my-messages/:identifier',
-  MyPageMyProfile = '/my-page/my-profile',
-  MyPageMyPersonalia = '/my-page/my-profile/my-personalia',
+  MyPageProfile = '/my-page/profile',
+  MyPageMyPersonalia = '/my-page/profile/personalia',
   MyPageMyProjectRegistrations = '/my-page/project-registrations/my-project-registrations',
-  MyPageMyProjects = '/my-page/my-profile/my-projects',
-  MyPageMyResearchProfile = '/my-page/research-profile/my-research-profile',
-  MyPageMyResults = '/my-page/my-profile/my-results',
+  MyPageMyProjects = '/my-page/profile/projects',
+  MyPageMyResearchProfile = '/my-page/profile/research-profile',
```

Comment:
ser du har fjerna for `MyPageMyProfile`, så kanskje ta det på resten også?
```suggestion
  MyPageProjects = '/my-page/profile/projects',
  MyPageResearchProfile = '/my-page/profile/research-profile',
```